spaces/S000172/README.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: S000172
+slug: kannan-rajagopalan-hart
+name: Kannan-Rajagopalan-Hart Space
+refs:
+  - mo: 434266
+    name: Answer to "Is there an infinite topological space with only countably many continuous functions to itself?"
+---
```

Comment:
I think it would be worth adding the Kannan-Rajagopalan doi:10.1016/0001-8708(78)90006-3 ("Constructions and applications of rigid spaces, I") to the refs.